package.json

```diff
@@ -155,6 +155,14 @@
           "default": "true",
           "description": "Mark ghost statements in the code (requires restart and Dafny 3.4.0+)"
         },
+        "dafny.dafnyPlugins": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "default": [],
+          "description": "Absolute paths to assemblies to be used as plugins (requires restart and Dafny 3.4.0+).\nExample 1: /user/home/dafnyplugin.dll\nExample 2: /user/home/dafnyplugin.dll,oneArgument\nExample 3: /user/home/dafnyplugin.dll,\"argument with space and \\\" escaped quote\" secondArgument"
```

Comment:
What happens if there's a comma in a plugin argument?  (Do we really need plugin arguments?)

Reply:
Only the first comma delimits the plugin from the argument list.
Yes we need plugin arguments, so that the same plugin can be configured through the class PluginConfiguration.
There are several use case for plugin arguments
* If the plugin provides several rewriters, to activate one of them only
* If rewriters have different options, such as "experimental" options, it makes it possible to activate them by changing a command-line flag.